rmf_site_format/src/camera_poses.rs

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
```

Comment:
Not sure if this file might have been copied from a long time ago, but I think this should be 2024.